packages/notification/package.json

```diff
@@ -5,10 +5,11 @@
   "author": "HashiCorp",
   "license": "MPL-2.0",
   "dependencies": {
+    "@hashicorp/flight-icons": "^2.0.2",
     "@hashicorp/platform-product-meta": "^0.1.0",
     "@hashicorp/react-close-button": "^0.1.1",
     "classnames": "^2.3.1",
-    "@hashicorp/flight-icons": "^2.0.2"
+    "framer-motion": "^6.3.11"
```

Comment:
So `framer-motion` kinda defacto becomes a dependency in the critical path as a result of this, along with its layout animation feature. I'm not sure the best way to handle this, but something to keep in mind. As soon as we start using this package in dev-portal our bundle will go up by like `~40kb`. Maybe we could try to use `m` instead of `motion` and use [LazyMotion](https://www.framer.com/docs/lazy-motion/) to include the features we need specifically.

Reply:
Introduced [`MotionConfig`](https://github.com/hashicorp/react-components/tree/main/packages/motion-config) component to manage lazy loading framer motions features.